src/lib/executors/Node.ts

```diff
@@ -757,7 +776,11 @@ export default class Node extends Executor<NodeEvents, Config, NodePlugins> {
       // handle suites
       this._instrumenter = createInstrumenter(
         Object.assign(
-          {},
+          {
+            esModules: Boolean(
+              typeof config.tsconfig === 'undefined' || config.tsconfig
```

Comment:
Should this depend on the `compilerOptions.module` property in the tsconfig? It shouldn't be necessary if, for example, the module style is "commonjs".

Reply:
I actually think right now this is not necessary...if the user wants to use es6 modules they can pass the instrumenterConfig themselves, and it's quite a pain to figure out what the `module` type would actually be from here, especially considering tsconfig extension.